### PR TITLE
docs: Improve docker setup instructions

### PR DESCRIPTION
**Pull request by Vitor-the-Dev**
## Description of change

Add a new step on Docker local setup instructions orienting the user to change to the project directory once they have Dockerfiles and Makefiles.

## Issues resolved by this PR

Eliminate possibility of misunderstanding arising from the fact only  "without docker" instructions oriented to cd into packages/sdks/javascript.

## More info

Post preview
pr_preview

### CODE REVIEW

**Comment by jfoliveira** [comment]
@cla-bot check

**Comment by jfoliveira** [comment]
> Thank you for your pull request and welcome to our community. We require contributors to sign our Contributor License Agreement, and we don't seem to have the users @Vitor-the-Dev on file. In order for us to review and merge your code, please contact the project maintainers to get yourself added.

Hello @Vitor-the-Dev! 👋
Welcome to Devopness open source projects!

The user `Vitor-the-Dev` has been added to our allowed contributors list.

Thank you 💖 and congrats 🎉 for opening your very first pull request in this repository.
Hope you have a great time contributing and learning around here! 😄

File: packages/sdks/javascript/README.md [hunk 1]
```diff
@@ -89,22 +89,27 @@ To build and test the SDK locally, [**fork this repository**](https://github.com
   xcode-select --install
   ```
 ### Setup and run in local environment
-#### 1. Build Docker Image
+#### 1. Navigate to the project directory!
```
Comment (jfoliveira) [inline packages/sdks/javascript/README.md:92]: I believe the exclamation mark here brings to the reader the idea of a very strong feeling being expressed.
As here we have a set of instruction with simple and clear steps, that might not be the most appropriate "voice tone" to be used here.

I'm offering a suggestion below to be quickly applied if you agree with it:
```suggestion
#### 1. Navigate to the project directory
```

**Review by jfoliveira — CHANGES_REQUESTED** [review]

File: packages/sdks/javascript/README.md (hunk 1 above)
Comment (Vitor-the-Dev) [reply packages/sdks/javascript/README.md:92]: Indeed, that is a very good point! I believe the same step on "Without docker" on the tutorial should also receive the same modifications.


```suggestion
#### 1. Navigate to the project directory
```

Comment (Vitor-the-Dev) [reply packages/sdks/javascript/README.md:92]: Indeed, that is a very good point! I believe the similar step on "Without docker" on the tutorial should also receive the same modifications.

Comment (jfoliveira) [reply packages/sdks/javascript/README.md]: @Vitor-the-Dev feel free to also modify the other step you mentioned if you feel it fits in this same PR.

**Review by jfoliveira — APPROVED** [review]

**Pull request merged**